charts/hivemq-platform-operator/crds/hivemq-platforms.hivemq.com-v1.yml

```diff
@@ -1,3 +1,4 @@
+# Generated by Fabric8 CRDGenerator, manual edits might get overwritten!
```

Comment:
if we don't update master branch of helm-chart with the CRD changes, the `hivemq-k8s-testsuite` repo might fail in the IT when comparing the CRDs, since at the moment this CRD will only be updated on the epic branch, and pipelines in the jenkins for other repos will use the master branch of the `helm-charts`, `hivemq-platform-operator`, and `hivemq-k8s-testsuite`.